src/web/components/Icon/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Icon({ children, color, size }) {
```

Comment:
Instead of using
```javascript
<Icon>fa-xxx</Icon>
```

why not to:

```javascript
<Icon name="fa-xxx" />
```

Reply:
I wanted to follow the logic of icon ligature that can be found in Material icon. However, the implementation of Bulma does not in fact follow this logic and my approach is surely incorrect. The approach you're proposing suits me very well because it's shorter.

I correct it at the next commit.